interfaces/builtin/jack.go

```diff
@@ -0,0 +1,47 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const jackSummary = `allows interacting with a JACK1 server`
+
+const jackBaseDeclarationSlots = `
+  jack1:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const jackConnectedPlugAppArmor = `
+# Per libjack/shm.c, various endpoints for JACK1 ae setup like:
+# jack-<userid>/<server name>/jack*
+owner /dev/shm/jack-[0-9]*/*/* rw,
+`
+
+func init() {
+	registerIface(&commonInterface{
+		name:                  "jack1",
+		summary:               jackSummary,
+		implicitOnClassic:     true,
+		baseDeclarationSlots:  jackBaseDeclarationSlots,
+		connectedPlugAppArmor: jackConnectedPlugAppArmor,
```

Comment:
Also adjust these to be jack1Summary, jack1BaseDeclarationSlots and jack1ConnectedPlugAppArmor.

Reply:
Done.